src/game/test/gameTimerTextBegin.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
```

Comment:
This comment is required for `jsdom` to work?

Reply:
yeah. It it necessary